workflow-service/src/main/java/com/redhat/parodos/workflow/version/WorkFlowVersionServiceImpl.java

```diff
@@ -33,6 +33,11 @@
 @Slf4j
 public class WorkFlowVersionServiceImpl implements WorkFlowVersionService {
 
+	public static String GetVersionHashForObject(Object workflowObject) throws IOException {
+		WorkFlowVersionServiceImpl versionService = new WorkFlowVersionServiceImpl();
```

Comment:
If you want to make it a Delegate/Util,  try to make the method "getHash" static  and remove "implements WorkFlowVersionService".